spark-hpcc/pom.xml

```diff
@@ -88,6 +88,13 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
will this dep apply to non-test builds as well?

Reply:
It will only apply to test builds due to the scope